compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__function_return_annotation_mismatch_with_try_nested.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: compiler-core/src/type_/tests/errors.rs
+expression: "fn() -> Result(Nil, Nil) {\n          try _ = {\n            try _ = {\n                try _ = Error(1)\n                Ok(Nil)\n            }\n            Ok(Nil)\n          }\n          Ok(Nil)\n        }"
+
+---
+error: Type mismatch
+  ┌─ /src/one/two.gleam:4:25
+  │  
+4 │                   try _ = Error(1)
+  │ ╭─────────────────────────^
+5 │ │                 Ok(Nil)
+6 │ │             }
+7 │ │             Ok(Nil)
+8 │ │           }
+9 │ │           Ok(Nil)
+  │ ╰─────────────────^
```

Comment:
I think it would be clearer to point to the start of the first try rather than the first value of the inner-most try. Could we do that instead?

Reply:
I remove the code to handle nested try and made it to point to the `try` keyword in all cases, is this ok ?

Reply:
I rebased `gleam-lang:main` and force pushed ee04f3a.